localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -85,8 +85,10 @@ services:
       TRUST_PROXY: ${TRUST_PROXY}
       AUTH_DATABASE_URL: postgresql://cloud_nine_wallet_auth:cloud_nine_wallet_auth@shared-database/cloud_nine_wallet_auth
       AUTH_SERVER_DOMAIN: ${CLOUD_NINE_AUTH_SERVER_DOMAIN:-http://localhost:3006}
+      REDIS_URL: redis://shared-redis:6379/0
```

Comment:
do we want to distinguish this from the `cloud-nine-backend` one?

Reply:
That's a good question. Not sure how much it matters TBH but I changed it to:

- c9 backend: 0
- c9 auth: 1
- hlb backend: 2
- hlb auth: 3